### ISSUE TITLE
Training from scratch.

### ISSUE THREAD

**Issue by make-j64** [aiff22/PyNET-Bokeh#11]
Hi @aiff22,
Im training from scratch with EBB dataset, result at train_iters 40000 of level 1 is wrong, like this:
https://ibb.co/bByGP4T

Is it normally?
Should i continue training to 100000 iters as your recommend?

Thanks.

**Comment by C-water** [comment]
I meet the same problem, the color of the image was distorted at level 1.

**Comment by C-water** [comment]
@unknow-dev  hi, have you found a solution? I tried to modify loss function into L1*100, and adjust the lr of level-1, but it did not work.   If you find the reason, could you share it with me?  Thank you.

**Comment by C-water** [comment]
@unknow-dev Hi, I recommend using the PyTorch code for training, and the author provides a basic version that requires modifications to level1 and level2. Unfortunately, the Tensorflow code I trained three times has gradient explosion that caused color distortion at level 1.

**Comment by make-j64** [comment]
> @unknow-dev Hi, I recommend using the PyTorch code for training, and the author provides a basic version that requires modifications to level1 and level2. Unfortunately, the Tensorflow code I trained three times has gradient explosion that caused color distortion at level 1.
Hi @chenquan-hdu ,
Thanks for your information, i will try.

**Comment by JialeHu97** [comment]
> @unknow-dev hi, have you found a solution? I tried to modify loss function into L1*100, and adjust the lr of level-1, but it did not work. If you find the reason, could you share it with me? Thank you.

@chenquan-hdu Hi, have you found a solution to the problem of image color distortion? If you find the solution, could you share it with me? Thank you!

**Comment by C-water** [comment]
Hi, 
I don't know how to solve this problem. Therefore, I used the PyTorch code for training, and the author provides a basic version (pynet-pytorch) that requires modifications to level1 and level2. PyTorch-version code is valid.

**Comment by JialeHu97** [comment]
@chenquan-hdu , Thanks for your information.
